runhouse/main.py

```diff
@@ -590,159 +592,149 @@ def status(
     _print_status(cluster_status, current_cluster)
 
 
-def get_clusters_from_den():
-    get_clusters_params = {"resource_type": "cluster", "folder": rns_client.username}
-    clusters_in_den_resp = rns_client.session.get(
-        f"{rns_client.api_server_url}/resource",
-        params=get_clusters_params,
-        headers=rns_client.request_headers(),
+def _create_output_table(
+    total_clusters: int,
+    running_clusters: int,
+    displayed_clusters: int,
+    filters_requested: bool,
+):
+    displayed_running_clusters = (
+        running_clusters
+        if running_clusters < displayed_clusters
+        else displayed_clusters
     )
+    table_title = f"[bold cyan]Clusters for {rns_client.username} (Running: {displayed_running_clusters}/{running_clusters}, Total Displayed: {displayed_clusters}/{total_clusters})[/bold cyan]"
 
-    return clusters_in_den_resp
+    table = Table(title=table_title)
 
+    if not filters_requested:
+        table.caption = f"[reset]Showing clusters that were active in the last {LAST_ACTIVE_AT_TIMEFRAME / HOUR} hours."
```

Comment:
```suggestion
        table.caption = f"[reset]Showing clusters that were active in the last{int(LAST_ACTIVE_AT_TIMEFRAME / HOUR)} hours."
```

nit - looks cleaner as an int if we're showing hours